workers/unity/Packages/com.improbable.gdk.mobile/Android/Utility/LaunchArguments.cs

```diff
@@ -8,6 +8,7 @@ public static class LaunchArguments
     {
         public static Dictionary<string, string> GetArguments()
         {
+#if !UNITY_EDITOR
```

Comment:
Was this malfunctioning in Unity Editor?

Reply:
Yes, as jamie and I were discussing in one of the previous threads in this PR: you can set your current platform to Android in the Editor, which will cause this code to be run. However using these java objects will only succeed in an actual android device/emulator. You end up with a nullpointer exception. 

So either
- we don‘t throw an exception
- we don‘t run this code in the editor

We decided that not running it seems like the better option, because this code shouldn‘t fail on an actual android device

Reply:
Would it be better to wrap this in `Application.isEditor` rather than compiling it out?